src/modules/secureStorage.json

```diff
@@ -0,0 +1,265 @@
+          "params": [
+            {
+              "name": "scope",
+              "value": "device"
+            },
+            {
+              "name": "key",
+              "value": "authRefreshToken"
+            },
+            {
+              "name": "value",
+              "value": "VGhpcyBub3QgYSByZWFsIHRva2VuLgo="
+            },
+            {
+              "name": "options",
+              "value": { 
+                "ttlvalue": 600
+              }
+            }
+          ],
+          "result": {
+            "name": "success",
+            "value": true
+          }
+        },
+        {
+          "name": "Set a refresh token with name authRefreshToken without optional parameter",
+          "params": [
+            {
+              "name": "scope",
+              "value": "account"
+            },
+            {
+              "name": "key",
+              "value": "authRefreshToken"
+            },
+            {
+              "name": "value",
+              "value": "VGhpcyBub3QgYSByZWFsIHRva2VuLgo="
+            }
+          ],
+          "result": {
+            "name": "success",
+            "value": true
+          }
+        }
+      ]
+    },
+    {
+      "name": "remove",
+      "summary": "Remove a secure data value",
+      "tags": [
+        {
+          "name": "capabilities",
+          "x-uses": [
+            "xrn:firebolt:capability:secure:storage"
+          ]
+        }
+      ],
+      "params": [
+        {
+          "name": "scope",
+          "summary": "The scope of the data key",
+          "schema": {
+            "$ref": "#/components/schemas/StorageScope"
+          },
+          "required": true
+        },
+        {
+          "name": "key",
+          "summary": "Key to remove",
+          "schema": {
+            "type": "string"
+          },
+          "required": true
+        }
+      ],
+      "result": {
+        "name": "success",
+        "summary": "",
+        "schema": {
+          "type": "boolean"
+        }
+      },
+      "examples": [
+        {
+          "name": "Remove the value with key authRefreshToken",
+          "params": [
+            {
+              "name": "scope",
+              "value": "device"
+            },
+            {
+              "name": "key",
+              "value": "authRefreshToken"
+            }
+          ],
+          "result": {
+            "name": "success",
+            "value": true
+          }
+        }
+      ]
+    }
+  ],
+  "components": {
+    "schemas": {
+      "StorageScope": {
+        "title": "StorageScope",
+        "type": "string",
+        "enum": [
+          "device",
+          "account"
+        ],
+        "description": "The scope of the data"
+      },
+      "StorageOptions": {
+        "title": "StorageOptions",
+        "type": "object",
+        "required": ["ttlvalue"],
+        "properties": {
+          "ttlvalue": {
+            "type": "number",
+            "description": "Value of the TTL"
```

Comment:
The description should have the units and what type of ttl it is. Like "Seconds from set time before the data expires and is removed"